include/simdjson/generic/ondemand/json_iterator-inl.h

```diff
@@ -191,6 +191,14 @@ inline std::string json_iterator::to_string() const noexcept {
           + std::string(" ]");
 }
 
+inline simdjson_result<const char *> json_iterator::current_location() noexcept {
+  auto cur_struct_index = token.position() - parser->implementation->structural_indexes.get();
+  if (cur_struct_index > static_cast<int>(parser->implementation->n_structural_indexes)) {
```

Comment:
This might be correct, but I was expecting `>=` and not merely `>`.

Reply:
Yes I think you might be right. Let me check!